packages/terra-form-input/src/FormInput.scss

```diff
@@ -0,0 +1,103 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
+    border-color: var(--terra-form-input-border-color, #4a4a4a);
+    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    border-style: var(--terra-form-input-border-style, solid);
+    border-width: var(--terra-form-input-border-width, 1px);
+    box-shadow: var(--terra-form-input-box-shadow, none);
+    color: var(--terra-form-input-color, #000);
+    font-family: inherit;
+    font-size: var(--terra-form-input-font-size, 1.143rem);
+    line-height: var(--terra-form-input-line-height, 1.143);
+    opacity: var(--terra-form-input-opacity, 1);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.571rem);
+    padding-left: var(--terra-form-input-padding-left, 0.357rem);
+    padding-right: var(--terra-form-input-padding-right, 0.357rem);
+    padding-top: var(--terra-form-input-padding-top, 0.571rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border-color: var(--terra-form-input-hover-border-color, #4a4a4a);
+      border-style: var(--terra-form-input-hover-border-style, solid);
+      border-width: var(--terra-form-input-hover-border-width, 1px);
+    }
+
+    &[required] {
+      box-shadow: unset;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border-color: var(--terra-form-input-disabled-border-color, #dedfe0);
+      border-style: var(--terra-form-input-disabled-border-style, solid);
+      border-width: var(--terra-form-input-disabled-border-width, 1px);
+      color: var(--terra-form-input-disabled-color, rgba(0, 0, 0, 0.25));
+      cursor: default;
+      opacity: var(--terra-form-input-disabled-opacity, 1);
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-focus-background-color, #fff);
+      border-color: var(--terra-form-input-focus-border-color, #4a4a4a);
+      border-radius: var(--terra-form-input-focus-border-radius, 0.2rem);
+      border-style: var(--terra-form-input-focus-border-style, solid);
```

Comment:
we'll need to merge the themable border styles into 1 shorthand border property for this @dylan-sftwr. It works around this bug: #1199
border-radius can be left stand-alone.

Reply:
Done. 5e9eba487a5de8d996a8f96c29aa6381196df310